Installing dimond fails due to boost errors

**Description**
I tried to install via pip but I got the following error even though boost is installed and BOOST_ROOT environment variable is set.
`dimod/roof_duality/src/fix_variables.cpp(30): fatal error C1083: Cannot open include file: 'boost/graph/graph_traits.hpp': No such file or directory`
Then I tried to install via source, but I got the same error.
This time I copied boost\boost_1_62_0\boost to dimod\dimod\roof_duality\src and it passed that stage, and after that it encounters a huge amount of the same syntax error, a sample of which you can see below:
![image](https://user-images.githubusercontent.com/11455681/96185737-36b72900-0f47-11eb-93c7-0c3486e12e03.png)

How can I get past all these errors? 

**Environment**
 - OS: Windows 10 x64
 - Python version: 3.9.0

--- a/setup.py
+++ b/setup.py
@@ -69,7 +69,7 @@ else:
 ext = '.pyx' if USE_CYTHON else '.cpp'
 
 extra_compile_args = {
-    'msvc': ['/std:c++latest', '/EHsc'],
+    'msvc': ['/std:c11', '/EHsc'],
     'unix': ['-std=c++11'],
 }
 

